src/models/room.ts

```diff
@@ -2344,6 +2344,14 @@ export class Room extends TypedEventEmitter<EmittedEvents, RoomEventHandlerMap>
         return this.getType() === RoomType.Space;
     }
 
+    /**
+     * Returns whether the room is a call-room as defined by MSC3417.
+     * @returns {boolean} true if the room's type is RoomType.Call
```

Comment:
I don't know why but this feels better here?

```suggestion
     * @returns {boolean} true if the room's type is m.call
```

Reply:
The room type isn't `m.call` at the moment, though - it's `org.matrix.msc3417.call`. Also, the `isSpaceRoom` docs above say the same thing.

Reply:
Ah, ok, _resolved_